Register-And-Login-Screen/app/src/main/java/com/example/login_register_screen_ui/Login.java

```diff
@@ -0,0 +1,17 @@
+package com.example.login_register_screen_ui;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.os.Bundle;
+import android.view.WindowManager;
+
+public class Login extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        getWindow().setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN,WindowManager.LayoutParams.FLAG_FULLSCREEN);
+        setContentView(R.layout.welcome_screen);
```

Comment:
Unnecessary multiple setContentView

[Android docs](https://developer.android.com/reference/android/app/Activity#setContentView%28android.view.View%29) state that setContentView will overwrite any existing view